src/NuGet.Services.BasicSearch/ConfigurationService.cs

```diff
@@ -35,27 +59,40 @@ public bool TryGet(string key, out string value)
                 {
                     var cloudKey = key.Replace(':', '-'); // no ':' supported in cloud services
                     value = SafeRoleEnvironment.GetConfigurationSettingValue(cloudKey);
-                    return true;
+                    setValue = true;
```

Comment:
Keeping the same behavior. If we are running in Azure, the RoleEnvironment configuration is the ultimate truth. 

---
In reply to: [71400834](https://github.com/NuGet/NuGet.Services.Metadata/pull/78#discussion_r71400834) [](ancestors = 71400834)